### PR TITLE
Fix: call parent method explicitly

### PR DESCRIPTION
**Pull request by liuzhenyilang**
Call parent method explicitly, make the code easier to understand.

### CODE REVIEW

File: worker/src/RTC/RtpStreamRecv.cpp [hunk 1]
```diff
@@ -843,7 +843,8 @@ namespace RTC
 		  score);
 #endif
 
-		RtpStream::UpdateScore(score);
+		// Call the parent method for update score
```
Comment (ibc) [inline worker/src/RTC/RtpStreamRecv.cpp]:
```suggestion
		// Call the parent method for update score.
```

File: worker/src/RTC/RtpStreamSend.cpp [hunk 2]
```diff
@@ -637,7 +637,8 @@ namespace RTC
 		  score);
 #endif
 
-		RtpStream::UpdateScore(score);
+		// Call the parent method for update score
```
Comment (ibc) [inline worker/src/RTC/RtpStreamSend.cpp]:
```suggestion
		// Call the parent method for update score.
```

**Review by ibc — COMMENTED** [review]
Please add dots at the end of comments as everywhere else.

**Comment by liuzhenyilang** [comment]
> Please add dots at the end of comments as everywhere else.

ok

**Comment by ibc** [comment]
NOTE: Dots not added yet.

**Comment by liuzhenyilang** [comment]
> NOTE: Dots not added yet.

Sorry, forget to commit.

**Pull request merged**